### PR TITLE
Add sidenav highlight var to sidenav guidance

### PR DESCRIPTION
**Pull request by thisisdano**
Added a line to the sidenav guidance to reference the `$sidenav-current-border-width` variable that now controls the highlight width.

### CODE REVIEW

File: _components/sidenav.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ lead: "Hierarchical, vertical navigation to place at the side of a page."
       <li>Indicate where a user is within the navigational hierarchy. Use the “active” state to show users which page they have navigated to.</li>
       <li>Keep the navigation links short. They can be shorter derivatives of page titles themselves.</li>
       <li>If the navigation hierarchy is too long, users may miss items at the bottom. If it’s too deep, users may miss items that require too many clicks. Usability test to find the right balance between breadth and depth.</li>
-      <li>The width of the current menu item's highlight border is set with the <code>$sidenav-current-border-width</code> variable.</li>
+      <li>The width of the current menu item‘s highlight border is set with the <code>$sidenav-current-border-width</code> variable.</li>
```
Comment (thisisdano) [inline _components/sidenav.md]: Thanks for changing this — but I've got to change it to the other direction! `’` instead of `‘`

Comment (thisisdano) [reply _components/sidenav.md]: `opt-shift-}`

Comment (thisisdano) [reply _components/sidenav.md]: Is there any way to use the `smartify` filter with our html content here?

File: _components/sidenav.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ lead: "Hierarchical, vertical navigation to place at the side of a page."
       <li>Indicate where a user is within the navigational hierarchy. Use the “active” state to show users which page they have navigated to.</li>
       <li>Keep the navigation links short. They can be shorter derivatives of page titles themselves.</li>
       <li>If the navigation hierarchy is too long, users may miss items at the bottom. If it’s too deep, users may miss items that require too many clicks. Usability test to find the right balance between breadth and depth.</li>
+      <li>The width of the current menu item’s highlight border is set with the <code>$sidenav-current-border-width</code> variable.</li>
```
Comment (maya) [inline _components/sidenav.md]: @thisisdano technical or developer info is put in the "Implementation" section. See this as an example: https://standards.usa.gov/components/buttons/

File: _components/sidenav.md (hunk 1 above)
Comment (maya) [reply _components/sidenav.md]: Could it be added here: https://github.com/18F/web-design-standards-docs/blob/master/_layouts/styleguide.html#L32 ?

**Comment by maya** [comment]
This can be merged once the next release is made, pushing https://github.com/18F/web-design-standards/pull/2202 into the next release.

**Pull request merged**